governance/xc_admin/packages/xc_admin_cli/src/index.ts

```diff
@@ -299,4 +305,59 @@ mutlisigCommand("approve", "Approve a transaction sitting in the multisig")
     await squad.approveTransaction(transaction);
   });
 
+mutlisigCommand("propose-token-transfer", "Propose token transfer")
+  .requiredOption("-a, --amount <number>", "amount in dollars")
+  .requiredOption("-d, --destination <pubkey>", "destination address")
+  .option(
+    "-m --mint <pubkey>",
+    "mint to transfer",
+    "EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v" // default value is USDC SPL
```

Comment:
Is it the same on both networks?

Reply:
I think this will only be used on mainnet